requirements.txt

```diff
@@ -1,3 +1,3 @@
-rubrix==0.7.0
```

Comment:
Oh, how did I manage to get such an old release ;)...

Reply:
:-)